src/components/Day.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import activityPropTypeShape from '../prop-types/activity';
```

Comment:
it would be nice to make an empty line to separate packages and relative imports.